lib/ret_web/templates/page/index-meta.html.eex

```diff
@@ -42,17 +53,20 @@
 <meta name="twitter:site" content="<%= @translations["app-twitter-user"] %>">
 <% end %>
 
-<meta name="twitter:title" content="<%= @translations["app-full-name"] || @translations["app-name"] %>">
+<meta name="twitter:title" content="<%= @name %>">
+<%= if @images["home_background"] do %>
 <meta name="twitter:image" content="<%= @images["home_background"] %>">
+<% end %>
 <meta name="twitter:description" content="<%= @description %>">
 
 <!-- Microsoft Edge -->
-<meta name="application-name" content="<%= @translations["app-full-name"] || @translations["app-name"] %>">
+<meta name="application-name" property="og:site_name" content="<%= @name %>">
+<meta name="creator" content="<%= @translations["company-name"] || @name %>" >
 <meta name="msapplication-tooltip" content="<%= @translations["app-tagline"] %>">
 <meta name="msapplication-starturl" content="/">
 <link rel="apple-touch-icon" href="/app-icon.png">
 
-<title><%= @translations["app-name"] %> - <%= @translations["app-tagline"] %></title>
+<title><%= @name %></title>
```

Comment:
The tagline is no longer included here now, and I think this was pretty much the only place where it is visible on a Hubs instance.  I think it should be included at least somewhere on the instance, and in the future it would be nice to add it to more places.

Reply:
Good catch!  I've added the app tagline back into the page title.